config/database.go

```diff
@@ -55,6 +55,7 @@ func NewDatabaseStore(dsn string) (ds *DatabaseStore, err error) {
 		db:             db,
 	}
 	if err = initializeConfigurationsTable(ds.db); err != nil {
+		ds.Close()
```

Comment:
Since `err` is a named return parameter, we can improve this by moving this to L50 and do a 

```
defer func() {
 if err != nil {
  db.Close()
 }
}()
```

And chance L59 

```
err = errors.Wrap(...)
return nil, err
```